client/src/components/Header.js

```diff
@@ -18,5 +33,14 @@ const Container = styled.div`
     text-align: center;
     color: white;
   }
+  section {
+    color: white;
+    text-align: center;
+    align: center;
+  }
+  Link {
```

Comment:
This really _shouldn't_ work.  It's not a valid CSS selector.  I'm still confused how it worked in that other implementation.

For this one, can you try styling the `a` element?  It should work better, and if it doesn't it's probably a specificity issue that can be solved reasonably.

Reply:
That worked ...you were right I was using the wrong Selector / or rather one that doesn't exist 👎

Reply:
💪